src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -8017,4 +8017,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_IteratorRefLikeElementType" xml:space="preserve">
     <value>Element type of an iterator may not be a ref struct or a type parameter allowing ref structs</value>
   </data>
+  <data name="IDS_FeatureNullConditionalAssignment" xml:space="preserve">
+    <value>null-conditional assignment</value>
```

Comment:
does this naming match our naming for `?.` and `?[...]`? Just want to be mostly consistent with that.

Reply:
The feature name for `?.` appears to be "null propagating operator".

However, the [standard](https://github.com/dotnet/csharpstandard/blob/draft-v8/standard/expressions.md#1288-null-conditional-member-access) uses the terms "null conditional member access" and "null conditional element access".

It feels like it is probably ok for pre-existing feature names to remain unchanged, while new feature names align with terms from the standard.

Reply:
matching the standard works for me.  i'd remove the `-` since it seems unnecesasry.